<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form_base3_cs</class>
 <widget class="QWidget" name="Form_base3_cs">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>462</width>
    <height>291</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item row="0" column="3">
       <widget class="QLineEdit" name="lineEdit_nummarche"/>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Delivery ref</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Society</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Start date</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="comboBox_society"/>
      </item>
      <item row="2" column="1">
       <widget class="QDateEdit" name="dateEdit_startdate">
        <property name="specialValueText">
         <string/>
        </property>
        <property name="minimumDateTime">
         <datetime>
          <hour>0</hour>
          <minute>0</minute>
          <second>0</second>
          <year>1752</year>
          <month>9</month>
          <day>15</day>
         </datetime>
        </property>
        <property name="minimumDate">
         <date>
          <year>1752</year>
          <month>9</month>
          <day>15</day>
         </date>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
        <property name="currentSectionIndex">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Estimated end date</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QDateEdit" name="dateEdit_enddate">
        <property name="specialValueText">
         <string/>
        </property>
        <property name="minimumDateTime">
         <datetime>
          <hour>0</hour>
          <minute>0</minute>
          <second>0</second>
          <year>1752</year>
          <month>9</month>
          <day>15</day>
         </datetime>
        </property>
        <property name="minimumDate">
         <date>
          <year>1752</year>
          <month>9</month>
          <day>15</day>
         </date>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
        <property name="currentSectionIndex">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit_nom"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_nom">
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
